include/Stand_der_Technik.tex

```diff
@@ -2,15 +2,18 @@ \chapter{Stand der Technik}
 \\
 \gls{CNN}’s trainieren eine Vielzahl von Filtern, die die gesuchten Features bestmöglich repräsentiert. Da dieses Produkt als Bild für Menschen jedoch keinen Sinn ergeben würde, wird es direkt mit einer Aktivierungsfunktion in eine Wahrscheinlichkeit oder eine Klassifizierung umgewandelt.\\
 Die Kaskadierung von \gls{CNN}’s ist eine Methode bei schrittweise komplexere und rechenintensivere \gls{CNN}’s nacheinander verwendet werden. Dies hat den Vorteil, dass Bildausschnitte die nur Hintergrund enthalten sehr schnell verworfen werden können. Um so wahrscheinlicher es ist einen Treffer zu haben um so bessere \gls{CNN}’s werden verwendet und so trotzdem eine sehr hohe Genauigkeit gewahrt.\\
 \\
-Um mittels Hough-Transformationen Formen zu erkennen wird zuerst mit einem Kantenerkennungsalgorithmus, wie z.B. der Canny-Algorithmus, alle Kanten in einem Binärbild repräsentiert. Im nächsten Schritt werden dann mittels Hough-Transformationen Formen wie Linien oder Kreise in diesem Binärbild identifiziert. Aus diesen Formen kann danach abgeleitet werden ob es sich um ein potenziell relevantes Objekt handelt.
-
-Weiter gibt es auch noch Methoden wie \gls{k-NN} oder \gls{SVM}. Diese werden aber immer weniger verwendet, da sie in nahezu jedem Anwendungsfall von neuronalen Netzen überholt wurden. \gls{k-NN} werden oft als erster einfacher versuch oder als simpler Algorithmus für einfache Problemstellungen verwendet, sowie in Kombination mit anderen Methoden. \gls{SVM}'s waren lange die besten Kandidaten in Objekterkennungs-Wettbewerben wie zum Beispiel ImageNet \parencite{ILSVRC15}. Bis die nötige Rechenleistung verfügbar war um Tiefe \gls{CNN} zu trainieren. Seither gewannen in solchen Challanges fast ausschliesslich \gls{CNN}.
+Um mittels Hough-Transformationen Formen zu erkennen wird zuerst mit einem Kantenerkennungsalgorithmus, wie z.B. der Canny-Algorithmus, alle Kanten in einem Binärbild repräsentiert. Im nächsten Schritt werden dann mittels Hough-Transformationen Formen wie Linien oder Kreise in diesem Binärbild identifiziert. Aus diesen Formen kann danach abgeleitet werden ob es sich um ein potenziell relevantes Objekt handelt.\\
```

Comment:
Um mittels Hough-Transformationen Formen zu erkennen, wird zuerst mit einem Kantenerkennungsalgorithmus, wie zum Beispiel dem Canny-Algorithmus, alle Kanten in einem Binärbild ~repräsentiert~ _gefunden/markiert/erkannt_.